internal/integration/integration_test.go

```diff
@@ -143,6 +144,21 @@ func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackMultiplePushFailMa
 	assert.Contains(suite.T(), outString, "remote: fatal: pack exceeds maximum allowed size")
 }
 
+func (suite *SpokesReceivePackTestSuite) TestSpokesReceivePackMultiplePushFailRefUpdateCommandLimit() {
+	assert.NoError(suite.T(), os.Chdir(suite.remoteRepo), "unable to chdir into our remote Git repo")
+	// Set a low value to receive.refupdatecommandlimit in order to make it fail
+	require.NoError(suite.T(), exec.Command("git", "config", "receive.refupdatecommandlimit", "1").Run())
+
+	assert.NoError(suite.T(), os.Chdir(suite.localRepo), "unable to chdir into our local Git repo")
+	out, err := exec.Command("git", "push", "--all", "--receive-pack=spokes-receive-pack-wrapper", "r").CombinedOutput()
```

Comment:
It took me a little bit to understand why this is failing (because we have 3 branches in this repo). It would be nicer to push them explicitly, e.g. `git push --receive-pack=spokes-receive-pack-wrapper r branch-1 branch-2` I think, that would make things slightly easier to understand.

Reply:
Fair enough! Added d1df5afaf888d02a46c93d6c9fe2eb02d8de73c4